styles/components/_buttons.scss

```diff
@@ -21,6 +21,7 @@
   &:hover {
     text-decoration: none;
     cursor: pointer;
+    border: solid 1px $focusedBorder;
```

Comment:
This should be border-color, otherwise any buttons with a different border width will get swapped to a 1px border width.